src/Activity/TrainWithLesson/RadioListLesson.js

```diff
@@ -6,14 +6,19 @@ import ListItemText from "@material-ui/core/ListItemText";
 import Radio from "@material-ui/core/Radio";
 import PropTypes from "prop-types";
 
-import lesson from "./Lesson";
+import lessonParallelograms from "./LessonParallelograms";
+import lessonMammals from "./LessonMammals";
```

Comment:
You forgot to add the two files `LessonParallelograms` and `LessonParallelograms`
When you add new files you should run the command `git add -A` or `git add <FILENAME>`